paystack-domain/src/main/java/com/chrisworks/paystackclient/domain/product/CreateProductRequest.java

```diff
@@ -0,0 +1,46 @@
+package com.chrisworks.paystackclient.domain.product;
+
+import com.chrisworks.paystackclient.domain.Amount;
+import com.chrisworks.paystackclient.domain.request.RequestBody;
+
+public class CreateProductRequest implements RequestBody<CreateProductRequest> {
+    private final String name;
+    private final String description;
+    private final String price;
+    private final String currency;
+    private final boolean unlimited;
+    private final Integer quantity;
```

Comment:
Make the `unlimited` and `quantity` fields optional, also use the builder pattern to declare setters for these fields. Compare what was done in the `UpdatePlanRequest.java` class

Reply:
Done